ko-KR.json

```diff
@@ -53,20 +53,20 @@
   "Change value is too large. Please try again with Token Fragmentation enabled from the Settings": "Change value is too large. Please try again with Token Fragmentation enabled from the Settings",
   "Choose your hardware wallet.": "하드웨어 지갑을 고르세요.",
   "Chromium only": "Chromium only",
-  "Circulating Supply": "Circulating Supply",
-  "Claim": "Claim",
-  "Claim {0} token": "Claim {0} token",
-  "Claimable Amount": "Claimable Amount",
-  "Clear ADAHandle": "Clear ADAHandle",
-  "Clear Filter": "Clear Filter",
+  "Circulating Supply": "유통량",
+  "Claim": "청구",
+  "Claim {0} token": "{0} 토큰 청구",
+  "Claimable Amount": "청구 가능한 수량",
+  "Clear ADAHandle": "ADAHandle 해제",
+  "Clear Filter": "필터 해제",
   "Close": "Close",
-  "Closed": "Closed",
+  "Closed": "마감",
   "Collateral input error. Please send yourself 5 ADA to cover for the collateral.": "Collateral input error. Please send yourself 5 ADA to cover for the collateral.",
   "Collective Zap In": "Collective Zap In",
-  "Collective Zap In (CZI) is a way for the community to zap into the pool with lower price impact and bonus yield farming boost at the end of the event.": "Collective Zap In (CZI) is a way for the community to zap into the pool with lower price impact and bonus yield farming boost at the end of the event.",
+  "Collective Zap In (CZI) is a way for the community to zap into the pool with lower price impact and bonus yield farming boost at the end of the event.": "CZI(Collective Zap In)는 이벤트가 끝날 때 가격 영향도를 낮추고 이자농사 부스트를 통해 커뮤니티가 풀에 참여할 수 있는 방법입니다.",.",
```

Comment:
there is an error in the end of this line 😄

Reply:
God..I will amend it and then pull request again! Sorry!